docs/addons.md

```diff
@@ -201,7 +201,7 @@ DESCRIPTION
 
 ## `heroku addons:plans SERVICE`
 
-list all available plans for an add-on services
+list all available plans for an add-on service
```

Comment:
Just a note for the future, this doc is autogenerated when we bump versions. This will be overridden by the description in the command itself. It doesn't hurt anything to change it now, but it's not necessary as long as the command description is correct.